### PR TITLE
Create identity.md

### PR DESCRIPTION
**Pull request by SavagePastaMan**
Tag to demonstrate the difference between `is` and `==`.

### CODE REVIEW

**Comment by vcokltfre** [comment]
For those curious here's what the tag looks like:
image

File: bot/resources/tags/identity.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+To check if two things are equal, use the equality operator (`==`).
+```py
+x = 5
+if x == 5:
+    print("x equals 5")
+if x == 3:
+    print("x equals 3")
+# Prints 'x equals 5'
+```
+
```
Comment (Akarys42) [inline bot/resources/tags/identity.md:14]: This empty line should be removed

**Review by Akarys42 — COMMENTED** [review]
I'm not a fan of the use of the word "thing". Maybe "object" would be a good replacement?

**Comment by mbaruh** [comment]
I wonder if this will result in questions about `None` and booleans. Maybe there should be a short note about that.

**Comment by Xithrius** [comment]
Author will not be able to complete tag. This PR is now up for grabs. Ask a core developer if you'd like to take over.

**Comment by vcokltfre** [comment]
Updated embed:
image

File: bot/resources/tags/identity.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+To check if two things are equal, use the equality operator (`==`).
+```py
+x = 5
+if x == 5:
+    print("x equals 5")
+if x == 3:
+    print("x equals 3")
+# Prints 'x equals 5'
+```
+To check if two things are actually the same thing in memory, use the identity comparison operator (`is`).
```
Comment (Akarys42) [inline bot/resources/tags/identity.md:14]: I still don’t like the use of `things` here. Perhaps `object` is a better word?

File: bot/resources/tags/identity.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+```py
+x = [1, 2, 3]
+y = [1, 2, 3]
+if x is [1, 2, 3]:
+    print("x is y")
+z = x
```
Comment (Akarys42) [inline bot/resources/tags/identity.md:20]: This snippet could get more explicit variable names too, it is a bit hard to follow right now. Also using the integer `420` would be less noisy than `[1, 2, 3]` and keep the same behavior.

**Review by Akarys42 — CHANGES_REQUESTED** [review]
Couple of things

File: bot/resources/tags/identity.md (hunk 3 above)
Comment (mbaruh) [reply bot/resources/tags/identity.md:20]: I think there's merit in using specifically lists, as it is something that confuses a lot of people.

For integers and strings this works only if the value is cached and shouldn't be relied on.

Comment (vcokltfre) [reply bot/resources/tags/identity.md]: I've kept it using the list given what zig said

**Review by Akarys42 — APPROVED** [review]
Lgtm, thanks!

**Review by Akarys42 — COMMENTED** [review]
.

**Review by Akarys42 — COMMENTED** [review]
Please review bot

**Review by mbaruh — APPROVED** [review]
